Source/iop/Iop_Dmac.h

```diff
@@ -67,6 +67,7 @@ namespace Iop
 		{
 			DPCR = 0x1F8010F0,
 			DPCR2 = 0x1F801570,
+			DPCR3 = 0xBF8015F0,

```

Comment:
Should be a physical address (ie.: `0x1F8015F0`)

Reply:
Oh boy, my bad